Makefile

```diff
@@ -128,16 +128,6 @@ define run_tests_in_docker
 	docker run --rm -ti -v `pwd`:/src -w /src "aiohttp-test-$(1)-$(2)" $(TEST_SPEC)
 endef
 
-.PHONY: test-3.9-no-extensions test
-test-3.9-no-extensions:
-	$(call run_tests_in_docker,3.9,y)
-test-3.9:
-	$(call run_tests_in_docker,3.9,n)
-test-3.10-no-extensions:
-	$(call run_tests_in_docker,3.10,y)
-test-3.10:
-	$(call run_tests_in_docker,3.10,n)
```

Comment:
Does this need to be removed?

Reply:
Possible not, but it seems unused and is confusing users.